src/frontend/src/routes/router.tsx

```diff
@@ -59,7 +60,7 @@ const Router = () => {
 
           <Route path="new" element={<PIAIntakeFormPage />}>
             <Route index element={<Navigate relative="path" to="intake" />} />
-            <Route path="intake" element={<PIAFormIntake />} />
+            <Route path="intake" element={<StoredPersonalInformation />} />
```

Comment:
```suggestion
            <Route path="intake" element={<PIAFormIntake />} />
```

Reply:
I noted this in the top of my PR. If you're putting here so that I don't forget then thanks.